docs/addons/calico-allow-burnham-riker.yaml

```diff
@@ -0,0 +1,21 @@
+# This GlobalNetworkPolicy uses Calico's CRD
```

Comment:
do we use these files anywhere in the project? Are they just samples? If so, are they redundant since you have them in docs?

Reply:
These are just samples, since I copied them straight into the doc, I can remove those two yaml files

Reply:
files are removed.